File: estruturas sequenciais/04.js
/*
fonte: https://wiki.python.org.br/EstruturaSequencial
Estruturas Sequenciais
04 - Faça um Programa que peça as 4 notas bimestrais e mostre a média
*/
const prompt = require('prompt-sync')({ sigint: true });

var nota_1 = parseInt(prompt("Digite a nota do primeiro bimestre: "));
var nota_2 = parseInt(prompt("Digite a nota do segundo bimestre: "));
var nota_3 = parseInt(prompt("Digite a nota do terceiro bimestre: "));
var nota_4 = parseInt(prompt("Digite a nota do quarto bimestre: "));

var media =  (nota_1 + nota_2 + nota_3 + nota_4) / 4

File: estruturas sequenciais/04.py
"""
fonte: https://wiki.python.org.br/EstruturaSequencial
Estruturas Sequenciais
4 - Faça um Programa que peça 4 notas bimestrais e mostre a média.
"""
import os

os.system('cls')
notas = []
n = 0
while len(notas) < 4:
    nota = input(f'Digite a {n + 1}° nota: ').strip().replace(",", ".")

    try:
        float(nota)
    except ValueError:
        print('digite um numero válido por favor')
    else:
        if 11 > float(nota) >= 0:
            notas.append(float(nota))
            n += 1
        else:
            print('digite um numero entre 0 e 10 por favor')

os.system('cls')
print(f"\nMedia das notas {notas} é: {sum(notas) / len(notas)}\n")
